packages/core/test/combinator/slice/index-test.js

```diff
@@ -27,14 +27,12 @@ describe('slice', function () {
 
     it('should narrow when second slice is smaller', function () {
       const s = slice(1, 5, slice(1, 10, now('')))
-      eq(2, s.min)
-      eq(6, s.max)
+      eq({ min: 2, max: 5 }, s.bounds)
```

Comment:
There is a changed behavior here because of how `mergeBounds`, used by `fuseSlice`, adds the minimums of the bounds, whereas the former implementation of `fuseSlice` calculated the fused bounds differently.

Could you please elaborate on this change?

Reply:
Good catch.  I need to look into this, but haven't had time yet.

Reply:
Yep, this is a bug in the PR 👍.  I'm adding failing test cases ...

Reply:
5989444 adds some exact tests and fixes the bug.  I also added a comment to try to clarify how the algorithm works.